tests/Js/Main/ListCollectorTests.fs

```diff
@@ -0,0 +1,26 @@
+module Fable.Tests.ListCollector
+
+open Util.Testing
+open Fable.Tests.Util
+open Microsoft.FSharp.Core.CompilerServices
+
+let cutOffLast list =
+    let mutable headList = ListCollector<'a>()
```

Comment:
Looking at the code it seems like we can remove the `mutable` here

Reply:
That would not reflect the actual usage of shared code. In regular F# the collector needs to be mutable.

Reply:
Didn't know about that.